src/js/setUpSite.js

```diff
@@ -236,11 +238,26 @@ const setUpSite = async () => {
   setUpAbout();
 
   const map = createMap();
-  await Promise.all([setUpCitiesLayer(map), setUpParkingLotsLayer(map)]);
+  await Promise.all([setUpCitiesLayer(map), setUpParkingLotsLayer(map), setUpSearch(initialCityId, "atlanta-ga")]);
 
   // There have been some issues on Safari with the map only rendering the top 20%
   // on the first page load. This is meant to address that.
   map.invalidateSize();
 };
 
+const setUpSearch = (initialCityId, fallbackCityId) => {
+  cities = {};
+  const element = document.getElementById("city-choice"); 
+    var choices = new Choices(element, {
+      maxItemCount: 1,
+      placeholder: true,
+      placeholderValue: "City Search",
+      itemSelectText: "Select",
+      searchEnabled: true,
+      position: "bottom",
+      allowHTML: true,
```

Comment:
This is a bad value. They plan to change the default to `false`: https://github.com/Choices-js/Choices

Reply:
Bump on this @Alankvuong